server/pkg/accounts/accounts.go

```diff
@@ -11,6 +11,13 @@ import (
 	"github.com/darwinfroese/hacksite/server/pkg/log"
 )
 
+// Account contains the information for each user
```

Comment:
Is there a reason this is commented out? We want this one added and the other one removed.